src/api/endpoints/auth/TokenManager.ts

```diff
@@ -17,7 +17,7 @@ export default class TokenManager extends ApiEndpoint {
     headerProvider?: HeaderProvider,
   ) {
     // todo: I dunno fix this someday I guess?
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+
```

Comment:
do we really not need this anymore? Sounds like we're missing one of the eslint plugins if this isn't creating an error?

Reply:
Didn't change that on my own since the lint:fix command removed it